ontology/catplus_ontology.ttl

```diff
@@ -789,6 +790,7 @@ cat:Sample a rdfs:Class, sh:NodeShape ;
                     sh:node cat:Observation ;
                     sh:node [sh:property    [sh:path qudt:unit ;
                                             sh:hasValue unit:PERCENT] ]];
+    sh:property [sh:path cat:hasPlate ; sh:class cat:Plate] ; # hasPlate

```

Comment:
on the Sample we need a `hasPlate` as well: usually the Samples are added from another plate, not the plate where most of the batch happens